packages/bpk-component-input/package-lock.json

```diff
@@ -1,17 +1,154 @@
 {
-	"requires": true,
+	"name": "bpk-component-input",
+	"version": "3.2.52",
 	"lockfileVersion": 1,
+	"requires": true,
 	"dependencies": {
 		"asap": {
 			"version": "2.0.6",
 			"resolved": "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz",
 			"integrity": "sha1-5QNHYR1+aQlDIIu9r+vLwvuGbUY="
 		},
+		"bpk-component-text": {
```

Comment:
You need to revert the changes from this lockfile - this happens when you do `npm install bpk-component-text --save-dev` inside the package directory. For `bpk-*` component that we want to be symlinked by lerna, we need to manually add the entry to `package.json` and run `npm run bootstrap` - subtle difference i know, but this bit me before.

Reply:
Done.